tests/assertions.py

```diff
@@ -178,7 +178,7 @@ def _assert_xarray_equal(left, right, check_attrs_values=True, check_attrs_order
     prefix = type(left).__name__
 
     # if decimals are supplied, do a rounding, otherwise rounding is just a dummy 'identity' func
-    rounding = lambda x: x if decimals is None else np.round(x, decimals=decimals)
+    rounding = lambda x: x if decimals is None else np.round(x, decimals=decimals)  # noqa: E731
```

Comment:
I think we should delete this function and just use the xr.testing functions instead.